api/test/e2enew/route/route_with_plugin_orchestration_test.go

```diff
@@ -14,100 +14,98 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package e2e
+package route
 
 import (
 	"io/ioutil"
 	"net/http"
-	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	"github.com/onsi/gomega"
+
+	"github.com/apisix/manager-api/test/e2enew/base"
 )
 
-func TestRoute_With_Plugin_Orchestration(t *testing.T) {
-	bytes, err := ioutil.ReadFile("../testdata/dag-conf.json")
-	assert.Nil(t, err)
+var _ = ginkgo.Describe("route with plugin orchestration", func() {
```

Comment:
Test cases here only check the admin API, but not concerned about the data use in DP, it's not a closed loop, we may request DP to verify DP's behaviors.

Reply:
@tokers 
It contains DP test, please see:
https://github.com/apache/apisix-dashboard/pull/1663/files#diff-b90858ef419a6b85e041becb1896e393f699a6e4c3c61ecd6b8822b3a2ffb22dR79-R88